### PR TITLE
Separate IC logic and wrapper

### PR DESCRIPTION
**Pull request by nminnie**
The base.py file is split into two files: IC logic is now in interactive_classifier.py, the Rodan wrapper is in wrapper.py.

### CODE REVIEW

File: rodan_job/interactive_classifier.py [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+        if glyph['id_state_manual']:
+            manual.append(glyph)
+
+    settings['class_names_json'] = serialize_class_names_to_json(settings)
+    settings['glyphs_json'] = serialize_glyphs_to_json(settings['glyphs'])
+    settings['training_json'] = serialize_glyphs_to_json(settings['training_glyphs'] + manual)
+
+
+# We don't want to reclassify glyphs that are a part of a group or split
+def filter_parts(settings):
```
Comment (vigliensoni) [inline rodan_job/interactive_classifier.py]: Add documentation string

File: rodan_job/interactive_classifier.py [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+    temp2 = copy.copy(settings['training_glyphs'])
+    for glyph in settings['training_glyphs']:
+        name = glyph['class_name']
+        if name.startswith("_split") or name.startswith("_group") or name.startswith("_delete"):
+            temp2.remove(glyph)
+    settings['training_glyphs'] = temp2
+    return parts
+
+
+def add_grouped_glyphs(settings):
```
Comment (vigliensoni) [inline rodan_job/interactive_classifier.py:335]: Add documentation string

**Review by vigliensoni — COMMENTED** [review]
Please do add missing documentation strings

**Comment by nminnie** [comment]
@vigliensoni 
I have added the documentation strings

**Review by vigliensoni — APPROVED** [review]

**Pull request merged**